collections/_article/maintenance-release-godot-3-5-2.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Maintenance release: Godot 3.5.2"
+excerpt: "We've just released Godot 4.0 after 3+ years of intense development, but we also still support the existing 3.5 stable branch. This maintenance release fixes a handful of issues which have been solved in the past few months, and could be backported to the 3.5 branch."
+categories: ["release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/maintenance-release-godot-3-5-2.jpg
+image_caption_title: Lumencraft
+image_caption_description: A game by 2Dynamic Games
+date: 2023-03-08 13:00:00
+---
+
+We've just released [Godot 4.0](/article/godot-4-0-sets-sail/) after 3+ years of intense development! This is the start of the Godot 4 journey, and there is still need for a few iterations for the 4.x branch to be as stable and reliable as the existing 3.5 branch.
+
+As such, it's natural that existing Godot 3.5 users would prefer to stay on that version for a while, and we're determined to provide them with bugfix releases. This 3.5.2 maintenance release fixes a handful of issues which have been solved in the past few months in parallel to Godot 4.0 development, and could be backported to the 3.5 branch. Nothing fancy, but a welcome upgrade from 3.5.1.
+
+**This is a safe and recommended update for all Godot 3.5.x users.** It should have no major incidence on your projects, even complex ones in production, if you're already using 3.5.1-stable.
```

Comment:
Is "incidence" a correct word here? I'm not sure looking at its definition if it makes sense.

Reply:
Frenchism it seems :D
![image](https://user-images.githubusercontent.com/4701338/223732674-ad31aa8d-213f-4026-b708-f375bf2744b5.png)

Reply:
Replaced by "impact".